content/legacy/_pub_2001_08_13_yapc-europe.md

```diff
@@ -52,4 +52,4 @@ The second edition of Yet Another Perl Conference in Europe was a great success.
 
 ------------------------------------------------------------------------
 
-Return to [Perl.com](http://perl.com)
+Return to [Perl.com](/)
```

Comment:
Let's just delete that line and remove the `---` above it. I made similar changes to another article last night.